CI/qt-silent-install.qs

```diff
@@ -27,7 +27,7 @@ Controller.prototype.ComponentSelectionPageCallback = function() {
     var widget = gui.currentPageWidget();
 
     widget.deselectAll();
-    widget.selectComponent("qt.qt5.51210.win32_mingw73");
+    widget.selectComponent("qt.qt5.5152.win32_mingw73");
```

Comment:
```suggestion
    widget.selectComponent("qt.qt5.5152.win32_mingw81");
```
I don't think GCC 7.3 is used any more - https://github.com/appveyor/ci/issues/3456 suggested checking https://ci.appveyor.com/project/appveyor-tests/qt-tests/builds/33896865 which only shows 8.1 for Qt 5.15.0 and presumable the later 5.15.2 that wasn't around I think when that was run.

Reply:
Makes sense